commit a6b6280c04078e12eb77351f8e818cb0bc9dfd80 (refs/changes/17/1817/2)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2017-04-20T23:11:09+00:00 (2 years, 6 months ago)
    
    MORAY-411 tools/rsync-to should not rely on moray being functional
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/tools/rsync-to b/tools/rsync-to
index 66d741a..62cbeca 100755
--- a/tools/rsync-to
+++ b/tools/rsync-to
@@ -20,9 +20,7 @@ TOP=$(cd $(dirname $0)/../; pwd)
 NODE=$1
 
 if [[ -z "$MORAY_ZONE" ]]; then
-    MORAY_ZONE=$(ssh $NODE "/opt/smartdc/bin/sdc-vmapi /vms" 2>/dev/null \
-        | json -H -c 'this.tags && this.tags.smartdc_role === "moray"' \
-            -c 'this.state === "running"' 0.uuid)
+    MORAY_ZONE=$(ssh $NODE "vmadm lookup -1 alias=moray0" 2>/dev/null)
 fi
 echo "MORAY_ZONE: $MORAY_ZONE"
 
